HQSmokeTests/testCases/test_01_others.py

```diff
@@ -7,6 +7,15 @@
 """"Contains all test cases that aren't specifically related any menu modules"""
 
 @pytest.mark.smoke
```

Comment:
what's the relevance of this tag? are there non-smoke tests in this suite?

Reply:
I kept it thinking that we might separate the export tests from the smoke runs as they keep failing. Maybe mark the exports with a different tag and execute them separately. But now the testcases are passing so I guess its safe to remove it.